<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Filter</class>
 <widget class="QDialog" name="Filter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filter</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../../svnroot/newsflash/gui/resource.qrc">
    <normaloff>:/resource/16x16_ico_png/ico_filter.png</normaloff>:/resource/16x16_ico_png/ico_filter.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAccept">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnApply">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Show items matching this criteria</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QGroupBox" name="grpAtLeast">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>Size is at least</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="0">
            <widget class="QSpinBox" name="spinAtLeast">
             <property name="maximum">
              <number>999999999</number>
             </property>
             <property name="value">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QSlider" name="sliderAtLeast">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>200</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="invertedControls">
              <bool>false</bool>
             </property>
             <property name="tickPosition">
              <enum>QSlider::NoTicks</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="chkKB">
            <property name="text">
             <string>KB</string>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="chkMB">
            <property name="text">
             <string>MB</string>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="chkGB">
            <property name="text">
             <string>GB</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="2">
         <widget class="QGroupBox" name="grpAtMost">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>Size is no more than</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="0">
            <widget class="QSpinBox" name="spinAtMost">
             <property name="maximum">
              <number>999999999</number>
             </property>
             <property name="value">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QSlider" name="sliderAtMost">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>200</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="invertedAppearance">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="grpAgeAtLeast">
          <property name="title">
           <string>Age is at least (days)</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QSlider" name="sliderAgeAtLeast">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinAgeAtLeast">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>99999</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QGroupBox" name="grpAgeAtMost">
          <property name="title">
           <string>Age is no more than (days)</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QSlider" name="sliderAgeAtMost">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinAgeAtMost">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>99999</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>grpAgeAtLeast</tabstop>
  <tabstop>sliderAgeAtLeast</tabstop>
  <tabstop>sliderAgeAtMost</tabstop>
  <tabstop>spinAgeAtLeast</tabstop>
  <tabstop>spinAgeAtMost</tabstop>
  <tabstop>btnAccept</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>btnApply</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../svnroot/newsflash/gui/resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>Filter</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
